### ISSUE TITLE
Can I use MindEye1's data to run this code?

### ISSUE THREAD

**Issue by Jamine-W** [MedARC-AI/MindEyeV2#39]
I have already downloaded MindEye1's data. Can I use MindEye1's data to run this code? What's the difference between MindEye1 and MindEye2's data processing?

**Comment by PaulScotti** [comment]
No, there is a substantial difference in the data (not in terms of the actual data points themselves, but in how the tar files get loaded). We changed the webdataset inputs to allow for more flexibility during testing, affording all the additional behavioral information in the "behav" array mentioned in the Readme
